### PR TITLE
Update c++quiz.md

### PR DESCRIPTION
**Pull request by VipulDhillon**
added a cpp question.

### CODE REVIEW

**Review by Ebazhanov — CHANGES_REQUESTED** [review]
merge conflicts

**Comment by Ebazhanov** [comment]
@VipulDhillon ping

**Comment by Ebazhanov** [comment]
@VipulDhillon pls fix merge conflicts

File: c++/c++quiz.md [hunk 1]
```diff
@@ -1436,3 +1436,10 @@ return 0;
 - [ ] `<limit>`
 - [x] `<limits>`
 - [ ] `<implementation>`
+
+#### Q91. Which statement is true?
```
Comment (auchynnikau) [inline c++/c++quiz.md:1440]: Duplicate

**Pull request closed without merging**